app/api/helpers/ICalExporter.py

```diff
@@ -48,7 +48,7 @@ def export(event_id):
             for speaker in session.speakers:
                 # Ref: http://icalendar.readthedocs.io/en/latest/usage.html#file-structure
                 # can use speaker.email below but privacy reasons
-                attendee = vCalAddress('MAILTO:' + event.email if event.email else 'undefined@email.com')
```

Comment:
That's not what we want to do. People will get emails of speakers by exporting large events and use it to create spam list

That's the privacy issue with it.

Ans also, there needs to a test

Reply:
Just to be clear, organizers are the only individuals who'd be able to export the events in ical formats right? Organizers would need the list of emails, wouldn't they?
Also, what email should be considered here? If we were to add an email attribute to the event, what should it be?

Reply:
If only organizers are able to export events, then it's not an issue.

If not, then owner email should be set

Reply:
Currently users who organize events are the only individuals who can export them in `.ical` format. Therefore, there doesn't seem to be an issue here. Will test it and remove `DO-NOT-MERGE`